src/lib.rs

```diff
@@ -17,7 +17,7 @@
 #![feature(alloc)]
 #![feature(const_fn, unique)]
 #![feature(associated_type_defaults)]
-#![feature(asm)]
+#![feature(asm,naked_functions, core_intrinsics)]
```

Comment:
Is there a reason why this feature is in a list and others aren't? Not an especially big issue. Also, there should be a space

Reply:
I grouped them together because I use them in the same place, but I can split them up.

Reply:
It's fine, just make sure the space is there